### PR TITLE
Fixing config generator in generators.md

### PR DESCRIPTION
**Pull request by americos**
This change fixes an issue where a `config.generators` block that shows how to create a new helper generator without stylesheets was missing `g.stylesheets  false`.

### CODE REVIEW

File: guides/source/generators.md [hunk 1]
```diff
@@ -214,6 +214,7 @@ The next customization on the workflow will be to stop generating stylesheet and
 config.generators do |g|
   g.orm             :active_record
   g.template_engine :erb
+  g.stylesheets     false
   g.test_framework  :test_unit, fixture: false
```
Comment (p8) [inline guides/source/generators.md:218]: A later example adds the stylesheets after the test_framework option.
So I think it makes sense do do the same here.
It also makes it easier to see the changes if we append options to the end.
```suggestion
  g.test_framework  :test_unit, fixture: false
  g.stylesheets     false
```

**Review by p8 — CHANGES_REQUESTED** [review]
Thanks for fixing this.
In the next paragraph we should also remove `JavaScript`, as that is still generated, I think.

**Comment by p8** [comment]
For documentation changes you can add `[ci-skip]` to the commit message to avoid running `buildkite/rails`.

**Comment by p8** [comment]
Can you squash your commits?

**Comment by americos** [comment]
> Can you squash your commits?

Sure thing, its done. Thanks for the feedback 👍

**Comment by p8** [comment]
Thanks @americos !

File: guides/source/generators.md [hunk 2]
```diff
@@ -215,10 +215,11 @@ config.generators do |g|
   g.orm             :active_record
   g.template_engine :erb
   g.test_framework  :test_unit, fixture: false
+  g.stylesheets     false
```
Comment (jonathanhefner) [inline guides/source/generators.md:218]: This was intentionally removed in https://github.com/rails/rails/commit/d5b9618da1ac60c674e8a27c3ab4e33742e9aa9b#diff-35420b0ec9d9ea5df03a3502ff68a23ed9c60ee2938a12a978bf75a9ed8264e7L231.  See https://github.com/rails/rails/pull/44490#discussion_r811368031.

**Pull request merged**